package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tab-split-2",
-  "description": "A tab-splitting extension for Firefox",
+  "description": "Open a mobile view of a page in the sidebar",
```

Comment:
Copied from manifest.json. I wasn't sure which one was right-er or better-er, or if they should have been left different. But realistically, package.json's value isn't used anywhere since this won't ever be published to npm. Happy to change it to whatever the consensus thinks.